packages/url_launcher/url_launcher_windows/windows/include/url_launcher_windows/url_launcher_plugin.h

```diff
@@ -4,7 +4,14 @@
 #ifndef PACKAGES_URL_LAUNCHER_URL_LAUNCHER_WINDOWS_WINDOWS_INCLUDE_URL_LAUNCHER_WINDOWS_URL_LAUNCHER_PLUGIN_H_
 #define PACKAGES_URL_LAUNCHER_URL_LAUNCHER_WINDOWS_WINDOWS_INCLUDE_URL_LAUNCHER_WINDOWS_URL_LAUNCHER_PLUGIN_H_
 
+#include <flutter/method_channel.h>
+#include <flutter/plugin_registrar_windows.h>
+#include <flutter/standard_method_codec.h>
 #include <flutter_plugin_registrar.h>
+#include <winrt/Windows.Foundation.h>
```

Comment:
Perhaps this is what's breaking Win32 compilation, if this header isn't available on the bots? I'm not clear on why this has been added in the first place; perhaps remove it and see what happens.

Reply:
Good point.. pushed a change to test this theory.